packages/venia-concept/src/queries/getCategoryList.graphql

```diff
@@ -3,9 +3,13 @@ query categoryList($id: Int!) {
     category(id: $id) {
         id
         children {
+            id
             name
             url_key
             url_path
+            product_count
```

Comment:
Are we using `product_count` anywhere? Can't find it in the source.

Reply:
I don't see it either. Removing it.